File: MY APPLICATION 29/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.LinearLayout;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity
{
    LinearLayout linearLayout;


    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        linearLayout=new LinearLayout(this);
        TextView txt1=new TextView(this);
        txt1.setText("Hello From Tops");
        int width=LinearLayout.LayoutParams.MATCH_PARENT;
        int height=LinearLayout.LayoutParams.WRAP_CONTENT;

        linearLayout.addView(txt1,width,height);
        setContentView(linearLayout);



    }
}

File: MY APPLICATION 5/main/java/com/example/myapplication/MainActivity.kt
package com.example.uiexample

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.LinearLayout
import android.widget.TextView

class MainActivity : AppCompatActivity()
{

    lateinit var linearLayout: LinearLayout

    override fun onCreate(savedInstanceState: Bundle?)
    {

        super.onCreate(savedInstanceState)
        //setContentView(R.layout.activity_table)

        linearLayout= LinearLayout(this)

        var txt1=TextView(this)
        var txt2=TextView(this)

        txt1.setText("Abcd")

        var width=500
        var height=250

        linearLayout.addView(txt1,width,height)

        setContentView(linearLayout)
    }
}